pennylane/transforms/metric_tensor.py

```diff
@@ -594,7 +591,7 @@ def _metric_tensor_hadamard(tape, allow_nonunitary, aux_wire, device_wires):
         block_sizes.append(len(layer_i.param_inds))
 
         for layer_j in layers[idx_i + 1 :]:
-            _tapes, _ids = _get_first_term_tapes(tape, layer_i, layer_j, allow_nonunitary, aux_wire)
+            _tapes, _ids = _get_first_term_tapes(layer_i, layer_j, allow_nonunitary, aux_wire)

```

Comment:
Why was tape removed from the call signature?

Reply:
it was only present to do `with tape.__class__() as new_tape:`, but it should just be AnnotatedQueue always. It's an internal function used only in this file so it's a-ok.